docs/README.md

```diff
@@ -0,0 +1,9 @@
+Table of contents
```

Comment:
Hey @alequetzalli, @derberg , @Florence-Njeri,

I have made this PR and added

Reply:
great start @Jagrutiti! thank you 🙏 

Let's have a quick call to help you with your questions and figure out the next steps. I will DM you now.